lib/java/desktop-environment/src/main/java/org/enso/desktopenvironment/WindowsTrashBin.java

```diff
@@ -0,0 +1,96 @@
+package org.enso.desktopenvironment;
+
+import java.nio.file.Path;
+import java.util.List;
+import org.graalvm.nativeimage.StackValue;
+import org.graalvm.nativeimage.c.CContext;
+import org.graalvm.nativeimage.c.constant.CConstant;
+import org.graalvm.nativeimage.c.function.CFunction;
+import org.graalvm.nativeimage.c.struct.CField;
+import org.graalvm.nativeimage.c.struct.CStruct;
+import org.graalvm.nativeimage.c.type.CCharPointer;
+import org.graalvm.nativeimage.c.type.CTypeConversion;
+import org.graalvm.word.PointerBase;
+import org.slf4j.LoggerFactory;
+
+@CContext(WindowsTrashBin.ShellApi.class)
```

Comment:
This `@CContext` & co. is called **System Java** and comes from `org.graalvm.nativeimage` API...